modules/virt-deleting-vmis-web.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-manage-vmis.adoc
+
+[id="virt-deleting-vmis-web_{context}"]
+= Deleting a standalone virtual machine instance using the web console
+
+
+.Procedure
+
+. In the {VirtProductName} console, click *Workloads* -> *Virtualization* from the side menu.
+
+. Click the &#8942; button of the standalone virtual machine instance (VMI) that you want to delete and select *Delete Virtual Machine Instance*.
+
+** Alternatively, click the standalone VMI name. The *Virtual Machine Instance Overview* page displays.
```

Comment:
Different folks might view this differently. I am sharing my thought. Changing the sentence to "standalone name of the VMI name" may come across a bit awkward. My .02 cents. Preferring to leave it as it is because of accuracy and ease of understanding.

Reply:
No, I was suggesting that you change it to "click the name of the VMI" or "click the name of the standalone VMI" if you want to specify that it's standalone.